src/ts/UIFactory.ts

```diff
@@ -356,11 +356,16 @@ function adsUILayout() {
             cssClasses: ['text-right'],
           }),
         ],
-        cssClasses: ['controlbar-top ad-controlbar-top'],
       }),
-      new AdControlBarBottom(),
+      new Container(<ContainerConfig>{
+        components: [
+          new PlaybackToggleButton(),
+          new VolumeToggleButton(),
+          new Spacer(),
+          new FullscreenToggleButton(),
+        ],
+      }),
```

Comment:
Please keep the previous approach, where it's possible to set CSS classes from here. This allows more flexibility in case someone wants to build his own UI layout. **Applies for all occasions**

```suggestion
      new Container({
        components: [
          new PlaybackToggleButton(),
          new VolumeToggleButton(),
          new Spacer(),
          new FullscreenToggleButton(),
        ],
        cssClasses: ['controlbar-top ad-controlbar-top'],
      }),
```

Reply:
Address in d2e834f0831c578144c93616a53a8527a5bdb30e